[Core] Non Unit Instance fractional value fix (#39293)
ray-project/ray: doc/source/ray-core/scheduling/resources.rst

    :start-after: __specifying_fractional_resource_requirements_start__
    :end-before: __specifying_fractional_resource_requirements_end__

.. note::

>   GPU, TPU, and Neuron Core requirements that are greater than 1, need to be whole numbers. For example, ``num_gpus=1.5`` is invalid.

Reviewer: ```suggestion
  GPU, TPU, and neuron_cores resource requirements that are greater than 1, need to be whole numbers. For example, ``num_gpus=1.5`` is invalid.
```

Reviewer: `neuron_cores` is the actual resource name here.

.. tip::

  Besides resource requirements, you can also specify an environment for a task or actor to run in,
  which can include Python packages, local files, environment variables, and more. See :ref:`Runtime Environments <runtime-environments>` for details.

.. note::

>   Unit resource requirements that are greater than 1, 

Author: the GPU, TPU and neuron core are written on the next line

  Besides resource requirements, you can also specify an environment for a task or actor to run in,
  which can include Python packages, local files, environment variables, and more---see :ref:`Runtime Environments <runtime-environments>` for details.

.. note::

>   Fractional input must be between 0 to 1. For any value greater than 1, `num_cpus` and `num_gpus` requires a whole number.

Reviewer: ```suggestion
  Fractional resource requirement  must be smaller than 1. For any resource requirement greater than 1, it needs to be a whole number (e.g. ``num_cpus=1.5`` is invalid).
```
Can you move it to be above tip?

>   For any unit resource (GPU, TPU, Neuron Core) requirement greater than 1, it needs to be a whole number (e.g. ``num_gpus=1.5`` is invalid).

Reviewer 2: can you add more details about what 's the definition of unit resource?

Reviewer 2: it is the internal terminology

  Besides resource requirements, you can also specify an environment for a task or actor to run in,

>   which can include Python packages, local files, environment variables, and more---see :ref:`Runtime Environments <runtime-environments>` for details.

Reviewer 3: ```suggestion
  which can include Python packages, local files, environment variables, and more. See :ref:`Runtime Environments <runtime-environments>` for details.
```

.. note::

>   For any unit resource (Resources assigned by scheduler using ids such as GPU, TPU, Neuron Core) requirement greater than 1, 

Reviewer 3: ```suggestion
  Unit resource requirements that are greater than 1, 
```

>   it needs to be a whole number (e.g. ``num_gpus=1.5`` is invalid).

Reviewer 3: ```suggestion
  need to be whole numbers. For example,  ``num_gpus=1.5`` is invalid. This restriction applies to resources that the scheduler assigns using IDs such as GPU, TPU, and Neuron Core.
```
